### PR TITLE
Update function support page for 0.9.0-beta

### PR DESCRIPTION
**Pull request by ronkok**
Update the function support page for the recent release.

This PR also changes the accordion sidebar from a JavaScript mechanism to a checkbox hack. I'm now getting better behavior on a touch screen when collapsing the accordion

### CODE REVIEW

File: function-support.html [hunk 1]
```diff
@@ -178,8 +178,22 @@
...
+#toc ul {
+	display: none;     /* by default hide all sub menus */
+}
+#toc label::after {
+   content: ' ▼';
+   font-size: 60%;
+}
+#toc input[type=checkbox]:checked + label + ul {
+	display: block;   /* show children when item is checked */
+}
```
Comment (kevinbarabash) [inline function-support.html]: cool

File: function-support.html [hunk 2]
```diff
@@ -178,8 +178,22 @@
...
+	/* hide native checkbox */
+	position: absolute;
+	opacity: 0;
+}
+
+#toc ul {
+	display: none;     /* by default hide all sub menus */
+}
+#toc label::after {
+   content: ' ▼';
```
Comment (kevinbarabash) [inline function-support.html]: It's a little weird to have a mix of spaces and tabs for indents, but I'm not too concerned about that for this file.

File: function-support.html [hunk 3]
```diff
@@ -761,11 +775,60 @@ <h3 id="other-letters">Other Letters</h3>
 			<td style="font-family: KaTeX_Main, KaTeX_AMS;">∂ ð ∇ ℑ ℓ ℘ ℜ Ⅎ ℵ ℶ ℷ ℸ ⅁</td>
 		</tr>
 		<tr>
-			<td colspan="2"><span style="font-family: Times New Roman;">ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞß<br>àáâãäåæçèéêëìíîïðñòóôöøùúûüýþÿ</span></td>
+			<td colspan="2"><span style="font-family: Times New Roman;">ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖÙÚÛÜÝÞß<br>àáâãäåçèéêëìíîïðñòóôöùúûüýþÿ</span></td>
```
Comment (kevinbarabash) [inline function-support.html]: Should also remove `Æ`.

File: function-support.html [hunk 4]
```diff
@@ -2896,38 +2981,14 @@ <h3>Links</h3>
 </div>
 
 <script>
-    setTreeToggleEvents(document.getElementById("toc"));  <!-- Table of Contents -->
-    renderMathInElement(document.body);                   <!-- KaTeX -->
+    renderMathInElement(document.body);    <!-- KaTeX -->
```
Comment (kevinbarabash) [inline function-support.html]: Nice!

File: function-support.html [hunk 5]
```diff
@@ -2896,38 +2981,14 @@ <h3>Links</h3>
 </div>
 
 <script>
-    setTreeToggleEvents(document.getElementById("toc"));  <!-- Table of Contents -->
-    renderMathInElement(document.body);                   <!-- KaTeX -->
+    renderMathInElement(document.body);    <!-- KaTeX -->
     document.getElementById("mobile-nav").addEventListener('click', toggleMobileMenu, false);
 
-    function setTreeToggleEvents(toc) {
```
Comment (kevinbarabash) [inline function-support.html]: Nice!

**Comment by kevinbarabash** [comment]
@ronkok thanks for keeping our function support page up to date.  :)

**Comment by ronkok** [comment]
You're very welcome. It would be sad if people would do all this great programming and then not tell anyone.

**Comment by edemaine** [comment]
Awesome work as usual, @ronkok.

Is the beta version on the demo page yet?

**Comment by ronkok** [comment]
Nope. I've never done that myself. I'm pretty content to stay with the function support page.

**Pull request merged**